tests/js/profile/wearable/widget/wearable/AssistPanel/AssistPanel.js

```diff
@@ -0,0 +1,286 @@
+				assert.ok(true, "_onSwipe on iherited widget was called");
+				assert.ok(!!event, "_onSwipe method has provided event object");
+				return true;
+			});
+			assistPanel.close = function () {
+				assert.ok(true, "called close method");
+			};
+
+			// tested method
+			assistPanel._onSwipe(event);
+
+			// restore stubs
+			helpers.restoreStub(tau.widget.core.Drawer.prototype, "_onSwipe");
+		});
+
+		/**
+		 * @type unit
+		 */
+		QUnit.test("_onSwipe (event has not direction down)", 2, function (assert) {
+			var assistPanel = new AssistPanel(),
+				event = {
+					detail: {
+						direction: "any"
+					}
+				};
+
+			// perpare stubs
+			helpers.stub(tau.widget.core.Drawer.prototype, "_onSwipe", function (event) {
+				assert.ok(true, "_onSwipe on iherited widget was called");
+				assert.ok(!!event, "_onSwipe method has provided event object");
+				return true;
+			});
+			assistPanel.close = function () {
+				assert.ok(false, "close method cannot be called when direction is different than 'down'");
+			};
+
+			// tested method
+			assistPanel._onSwipe(event);
+
+			// restore stubs
+			helpers.restoreStub(tau.widget.core.Drawer.prototype, "_onSwipe");
+		});
+
+		/**
+		 * @type unit
+		 */
+		QUnit.test("_bindEvents", 2, function (assert) {
+			var assistPanel = new AssistPanel(),
+				onClick = function () {
+					// noop
+				};
+
+			// perpare stubs
+			helpers.stub(tau.widget.core.Drawer.prototype, "_bindEvents", function () {
+				assert.ok(true, "_bindEvents on iherited widget was called");
+				return true;
+			});
+			assistPanel._ui.indicator = document.createElement("div");
+			assistPanel._callbacks.onClick = onClick;
+
+			// tested method
+			assistPanel._bindEvents();
+
+			assert.notEqual(assistPanel._callbacks.onClick, onClick, "new onClick handler has been added");
+
+			// restore stubs
+			helpers.restoreStub(tau.widget.core.Drawer.prototype, "_bindEvents");
+		});
+
+		/**
+		 * @type unit
+		 */
+		QUnit.test("_unbindEvents", 2, function (assert) {
+			var assistPanel = new AssistPanel(),
+				onClick = function () {
+					// noop
+				};
+
+			// perpare stubs
+			assistPanel._callbacks.onClick = onClick;
+			assistPanel._ui = {
+				indicator: {
+					removeEventListener: function (eventName, callback) {
+						assert.equal(eventName, "vclick", "removeEventListener has been called");
+						assert.equal(callback, onClick, "rmoved properly event handler");
+					}
+				}
+			}
+
+			// tested method
+			assistPanel._unbindEvents();
+		});
+
+		/**
+		 * @type unit
+		 */
+		QUnit.test("_destroy", 2, function (assert) {
+			var assistPanel;
+
+			// perpare stubs
+			helpers.stub(tau.widget.core.Drawer.prototype, "_destroy", function () {
+				assert.ok(true, "_destroy on iherited widget was called");
+				return true;
+			});
+			// create widget instance
+			assistPanel = new AssistPanel();
+
+			assistPanel._unbindEvents = function () {
```

Comment:
Do we need this call to _unbindEvents here? Looks like it's optional call since we are testing destroy method.

Reply:
we have to verify _unbind method too. It isn't optional.